pkg/util/file_indexer_test.go

```diff
@@ -0,0 +1,129 @@
+package util
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/openshift/odo/pkg/testingutil/filesystem"
+)
+
+func TestCheckGitIgnoreFile(t *testing.T) {
+
+	// create a fake fs in memory
+	fs := filesystem.NewFakeFs()
+	// create a context directory on fake fs
+	contextDir, err := fs.TempDir(os.TempDir(), "context")
+	if err != nil {
+		t.Error(err)
+	}
+
+	gitignorePath := filepath.Join(contextDir, ".gitignore")
+
+	tests := []struct {
+		testName        string
```

Comment:
We use `name` in the unit tests I have come across.

Reply:
I think we use both ```name``` as well ```testName```. As I came cross in https://github.com/openshift/odo/blob/master/pkg/util/util_test.go#L22 

We have used both in our unit test. Let me know your views on this.

Reply:
Haha. Fine by me. :smile:

Reply:
Cool 👍